lib/channels_list.rb

```diff
@@ -58,7 +58,8 @@ module ChannelsList
       'pairing_notifications': 'C29J3DPGW',
       'standup_notifications': 'C29JE6HGR',
       'premium_extra': 'C29J4QQ9M',
-      'premium_mob_trialists': 'C29J4QQ9F'
+      'premium_mob_trialists': 'C29J4QQ9F',
+      'new-members': 'C02G8J699'
```

Comment:
Is there a plan to move these to the database?

Reply:
yes! 
we have already put in a `PR` that has been merged to `master`, which uses the projects slack channels join table... we create a rake task to go through this hard coded channels to the database.
After this has made its way to production, I will be adding some channels manually to the events slack channels join table, then I'll put in a third `PR` to allow users to add the slack_channels to the event themselves through the interface.